src/openms/include/OpenMS/FORMAT/ParamCTDFile.h

```diff
@@ -0,0 +1,113 @@
+// --------------------------------------------------------------------------
+//                   OpenMS -- Open-Source Mass Spectrometry
+// --------------------------------------------------------------------------
+// Copyright The OpenMS Team -- Eberhard Karls University Tuebingen,
+// ETH Zurich, and Freie Universitaet Berlin 2002-2021.
+//
+// This software is released under a three-clause BSD license:
+//  * Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+//  * Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//  * Neither the name of any author or any participating institution
+//    may be used to endorse or promote products derived from this software
+//    without specific prior written permission.
+// For a full list of authors, refer to the file AUTHORS.
+// --------------------------------------------------------------------------
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL ANY OF THE AUTHORS OR THE CONTRIBUTING
+// INSTITUTIONS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+// OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+// WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+// OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+// ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// --------------------------------------------------------------------------
+// $Maintainer: Ruben Grünberg $
+// $Authors: Ruben Grünberg $
+// --------------------------------------------------------------------------
+
+#pragma once
+
+#include <string>
+#include <OpenMS/DATASTRUCTURES/Param.h>
+
+namespace OpenMS
+{
+
+  /**
+   @brief A struct to pass information about the tool as one parameter
+   */
+  struct ToolInfo
+  {
+    std::string version_;
+    std::string name_;
+    std::string docurl_;
+    std::string category_;
+    std::string description_;
+    std::string openms_doi_;
+    std::vector<std::string> citations_;
+  };
+
+  /**
+  @brief Serializes a Param class in paramCTD file format.
+         Note: only storing is currently possible
+
+*/
+  class OPENMS_DLLAPI ParamCTDFile
+  {
+  public:
+    ParamCTDFile(); ///Constructor
```

Comment:
better: make it "= default"
and default initialize strings in header file
 const std::string schema_location_="...";